### PR TITLE
Admin client

### PR DESCRIPTION
**Pull request by ods**
### Changes

Based on https://github.com/aio-libs/aiokafka/pull/639

### Checklist

- [x] I think the code is well written
- [x] Unit tests for the changes exist
- [ ] Documentation reflects the changes
- [x] Add a new news fragment into the `CHANGES` folder
  * name it `<issue_id>.<type>` (e.g. `588.bugfix`)
  * if you don't have an `issue_id` change it to the pr id after creating the PR
  * ensure type is one of the following:
    * `.feature`: Signifying a new feature.
    * `.bugfix`: Signifying a bug fix.
    * `.doc`: Signifying a documentation improvement.
    * `.removal`: Signifying a deprecation or removal of public API.
    * `.misc`: A ticket has been closed, but it is not of interest to users.
  * Make sure to use full sentences with correct case and punctuation, for example: `Fix issue with non-ascii contents in doctest text files.`

### CODE REVIEW

File: tests/test_admin.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    async def test_describe_configs(self):
+        tn = self.topic_name
+        await self.admin.create_topics([NewTopic(tn, 1, 1)])
+        cr = ConfigResource(ConfigResourceType.TOPIC, tn, None)
+        resp = await self.admin.describe_configs([cr])
+        assert len(resp) == 1
+        assert len(resp[0].resources) == 1
+        config_resource = resp[0].resources[0]
+        assert config_resource[3] == tn
+        assert False, config_resource
```
Comment (ods) [inline tests/test_admin.py:75]: Looks like unfinished

File: tests/test_admin.py [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+        assert resp.topic_errors is not None
+        assert len(resp.topic_errors) == 1
+        topic, error_code, error = resp.topic_errors[0]
+        assert topic == name
+        assert error_code == 0
+        assert not error
+
+    @run_until_complete
+    async def test_list_topics(self):
+        topic_names = {self.topic_name, self.topic_name}
```
Comment (ods) [inline tests/test_admin.py:42]: It's not obvious that property gives new value each time

File: tests/test_admin.py [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+        topics = [NewTopic(tn, 1, 1) for tn in topic_names]
+        await self.admin.create_topics(topics)
+        actual = await self.admin.list_topics()
+        for tn in topic_names:
+            assert tn in actual
```
Comment (ods) [inline tests/test_admin.py:47]: It's better done with either `==` or `<=` (subset) check for the whole list

File: tests/test_admin.py [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+        assert len(old_desc[0]["partitions"]) == 1
+        new_part = NewPartitions(total_count=2)
+        await self.admin.create_partitions({tn: new_part})
+        new_desc = await self.admin.describe_topics([tn])
+        assert len(new_desc[0]["partitions"]) == 2
+        assert new_desc[0]["partitions"][0] == old_desc[0]["partitions"][0]
+
+    @run_until_complete
+    async def test_list_consumer_groups(self):
+        resp = await self.admin.list_consumer_groups()
```
Comment (ods) [inline tests/test_admin.py:104]: TODO: Create at least one consumer

**Review by tvoinarovskyi — APPROVED** [review]

**Pull request merged**